pombola/south_africa/templatetags/za_election_list_links.py

```diff
@@ -0,0 +1,38 @@
+from django import template
+
+register = template.Library()
+
+@register.filter()
+def get_party_slug(org_slug):
+    """Takes the original slug of an election list 'organisation' and returns
+    the party slug
+    """
+
+    stripped = strip_slug(org_slug)
+    parts = stripped.split("--")
+    return parts[0]
+
+@register.filter()
+def get_place_slug(org_slug):
+    """Takes the original slug of an election list 'organisation' and returns
+    the place (province) slug
+    """
+
+    stripped = strip_slug(org_slug)
+    parts = stripped.split("--")
+    return parts[1]
+
+def strip_slug(slug):
```

Comment:
I think I'd find this easier to have confidence in if it was picked apart with a regular expression, e.g. in this module you could add:

```python
election_list_slug_re = re.compile(
    r'''^
    (?P<party>.*)-
    (?P<place_type>provincial|national)-
    (?P<place>.*)-
    election-list-
    (?P<year>.*)
    $''',
    re.VERBOSE
)
```

... which seems like a clearer way of decomposing the slug. Then you can do:

```python
m = election_list_re.search(org_slug)
```

... and either raise an exception if `m` is `None` or return `m.groupdict()` otherwise. The dict will make the other template filter functions read better as well.